### PR TITLE
Div

### PR DESCRIPTION
**Pull request by shwky56**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -144,536 +144,11 @@ <h1>New to Udemy? Lucky you.</h1>
...
-            <span class="fa fa-star checked"></span>
-            <span class="fa fa-star checked"></span>
-            <span class="fa fa-star checked"></span>
-            <span class="fa fa-star"></span>
-          </article>
-        </a>
-      </div>
+    </div>
       </section>
+      <div class="tist" id="tist"></div>
```
Comment (ahmed2m) [inline index.html]: This div serves no purpose, right?

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    val = val.toLowerCase();
+    let newcourses = await getCourses();
+    newcourses = newcourses.filter(course => {
+        let string = course.title;
+        string = string.toLowerCase();
+        return string.includes(val);
+    });
+    renderCourses(newcourses);
+} 
+async function cearch(val){
```
Comment (ahmed2m) [inline js/main.js]: I think this is a debugging function? if so it should be removed

**Review by ahmed2m — CHANGES_REQUESTED** [review]
Great job overall, two minor comments!

**Comment by shwky56** [comment]
Yes, this is correct. I will fix these, and thank you

‫في الثلاثاء، 23 أغسطس 2022 في 12:05 م تمت كتابة ما يلي بواسطة ‪Ahmed
Mohamedeen‬‏ ***@***.***‬‏>:‬

**Review by ahmed2m — APPROVED** [review]
Great work!

**Pull request merged**